sql/migrations/20180723122149_world.sql

```diff
@@ -0,0 +1,32 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20180723122149');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20180723122149');
+-- Add your query below.
+
+UPDATE `creature_template` SET `minhealth`='22272', `maxhealth`='22272', `armor`='3380', `speed_walk`='1', `mindmg`='88.1', `maxdmg`='116.9', `attackpower`='238', `dmg_multiplier`='6.5', `baseattacktime`='1800', `rangeattacktime`='1800' WHERE `entry`='9033';
+UPDATE `creature` SET `curhealth`='22272' WHERE `id`='9033';
+
+REPLACE INTO `script_texts` (`entry`, `content_default`, `sound`, `type`, `language`, `emote`, `comment`) VALUES
+('-1230004', '%s cries out an alarm!', '0', '2', '0', '0', 'angerforge EMOTE_ALARM');
```

Comment:
Do not add anything to `script_texts`, this table is in the process of being deprecated. Find the text in `broadcast_text`, it contains all blizzlike texts.

Reply:
sure